lib/src/showcase.dart

```diff
@@ -296,7 +300,9 @@ class Showcase extends StatefulWidget {
         assert(onTargetClick == null || disposeOnTap != null,
             "disposeOnTap is required if you're using onTargetClick"),
         assert(disposeOnTap == null || onTargetClick != null,
-            "onTargetClick is required if you're using disposeOnTap");
+            "onTargetClick is required if you're using disposeOnTap"),
+        assert(onBarrierClick == null || disableBarrierInteraction == false,
+            "can't use both property at same time");
```

Comment:
please include both properties' names instead of only mentioning "both" in the assert message.